server/webapp/WEB-INF/rails/webpack/views/pages/pipelines/spec/template_editor_spec.tsx

```diff
@@ -60,6 +60,20 @@ describe("AddPipeline: TemplateEditor", () => {
     expect(isUsingTemplate()).toBe(false);
     expect(config.template() === undefined).toBeTruthy();
   });
+
+  it("should not display dropdown and should display flash when cache fails to populate", () => {
+    helper.unmount();
+    helper.mount(() => <TemplateEditor pipelineConfig={config} isUsingTemplate={isUsingTemplate} cache={new FailedTestCache()}/>);
+    helper.click(helper.byTestId("switch-paddle"));
```

Comment:
There's also `helper.clickByDataTestId()` which does just this. Not a problem, just letting you know.